### ISSUE TITLE
Investigate distributing a non-WASM build

### ISSUE THREAD

**Issue by mdaines** [mdaines/viz.js#182]
It's not possible to use WASM in some environments. For example, because of a [Content Security Policy](https://www.w3.org/TR/CSP3/#wasm-integration).

Emscripten can build a JavaScript-only version of Viz.js by using the setting `WASM=0`, so this seems to be mainly a matter of how to structure the build, and publish such a package.

**Comment by lborgman** [comment]
Did you try this?

**Comment by mdaines** [comment]
@lborgman Yes, thank you for reminding me. I took another look at this recently.

Emscripten has a setting that will output a fallback JS file for browsers than don't support WebAssembly (`WASM=2`). I was able to get Graphviz/Viz.js to build using this setting and successfully render graphs with the fallback.

It seems like a reasonable option, but I need to sort out how it would be packaged.
